// bootstrap-vue-next/dist/bootstrap-vue-next.mjs
import { i } from "./index-BjYm4lhC.mjs";
import { i as i2 } from "./index-CXpBEYSc.mjs";
import { i as i3 } from "./index-i-UFv70c.mjs";
import { i as i4 } from "./index-BggLBiUX.mjs";
import { c as componentNames, a as componentsWithExternalPath, b as composableNames, d as composablesWithExternalPath, e as directiveNames, f as directivesWithExternalPath } from "./index-sLCKmIWG.mjs";
import { B, i as i5 } from "./index-sLCKmIWG.mjs";
import { i as i6 } from "./index-DwDHvBIQ.mjs";
import { _, a } from "./BAccordionItem.vue_vue_type_script_setup_true_lang-GcHvVqGo.mjs";
import { _ as _2, a as a2 } from "./BAvatarGroup.vue_vue_type_script_setup_true_lang-Bh51NGgl.mjs";
import { _ as _3 } from "./BBadge.vue_vue_type_script_setup_true_lang-paO8jjw3.mjs";
import { _ as _4, a as a3 } from "./BBreadcrumb.vue_vue_type_script_setup_true_lang-Byv4zD_-.mjs";
import { _ as _5 } from "./BButton.vue_vue_type_script_setup_true_lang-D1hoIq9T.mjs";
import { _ as _6, a as a4 } from "./BButtonToolbar.vue_vue_type_script_setup_true_lang-D60e-V3m.mjs";
import { _ as _7 } from "./BCloseButton.vue_vue_type_script_setup_true_lang-C9vJ2REQ.mjs";
import { _ as _8, a as a5, b, c, d, e, f } from "./BCard.vue_vue_type_script_setup_true_lang-CkPaeN_W.mjs";
import { _ as _9, a as a6 } from "./BCardText.vue_vue_type_script_setup_true_lang-CJg09Wfj.mjs";
import { _ as _10, a as a7 } from "./BCarouselSlide.vue_vue_type_script_setup_true_lang-C4M0nfcE.mjs";
import { _ as _11 } from "./BCollapse.vue_vue_type_script_setup_true_lang-CkafXUNX.mjs";
import { _ as _12 } from "./BCol.vue_vue_type_script_setup_true_lang-COPhxQ7s.mjs";
import { _ as _13 } from "./BContainer.vue_vue_type_script_setup_true_lang-DmmLoJOw.mjs";
import { _ as _14 } from "./BRow.vue_vue_type_script_setup_true_lang-BjssnBKB.mjs";
import { _ as _15, a as a8, b as b2, c as c2, d as d2, e as e2, f as f2 } from "./BDropdownText.vue_vue_type_script_setup_true_lang-mCfnTsWh.mjs";
import { _ as _16 } from "./BForm.vue_vue_type_script_setup_true_lang-BNMZGXOg.mjs";
import { _ as _17, a as a9 } from "./BFormFloatingLabel.vue_vue_type_script_setup_true_lang-u01tzxQ5.mjs";
import { _ as _18, a as a10, b as b3, c as c3 } from "./BFormValidFeedback.vue_vue_type_script_setup_true_lang-Dqfnau41.mjs";
import { _ as _19, a as a11 } from "./BFormCheckboxGroup.vue_vue_type_script_setup_true_lang-CqPRpPKk.mjs";
import { _ as _20 } from "./BFormFile.vue_vue_type_script_setup_true_lang-DU35VHX7.mjs";
import { _ as _21 } from "./BFormGroup.vue_vue_type_script_setup_true_lang-J35v3PhC.mjs";
import { _ as _22, _ as _23 } from "./BFormInput.vue_vue_type_script_setup_true_lang-BFYwAuBb.mjs";
import { _ as _24, a as a12 } from "./BFormRadioGroup.vue_vue_type_script_setup_true_lang-CGgu9vhJ.mjs";
import { _ as _25, a as a13 } from "./BFormSelect.vue_vue_type_script_setup_true_lang-DGTlXd8-.mjs";
import { _ as _26 } from "./useFormSelect-NtZjej0I.mjs";
import { _ as _27 } from "./BFormSpinbutton.vue_vue_type_script_setup_true_lang-BKR-rLKd.mjs";
import { a as a14, _ as _28 } from "./BFormTags.vue_vue_type_script_setup_true_lang-Bx4ftUiU.mjs";
import { _ as _29 } from "./BFormTextarea.vue_vue_type_script_setup_true_lang-DmHZamgx.mjs";
import { _ as _30 } from "./BImg.vue_vue_type_script_setup_true_lang-6-d-9mye.mjs";
import { _ as _31, a as a15 } from "./BInputGroupText.vue_vue_type_script_setup_true_lang-BE8W-jh2.mjs";
import { _ as _32 } from "./BLink.vue_vue_type_script_setup_true_lang-CRL42bxc.mjs";
import { _ as _33, a as a16 } from "./BListGroupItem.vue_vue_type_script_setup_true_lang-EHl_hWY1.mjs";
import { B as B2, _ as _34 } from "./BModalOrchestrator.vue_vue_type_script_setup_true_lang-BulTQr4i.mjs";
import { _ as _35, a as a17, b as b4, c as c4, d as d3 } from "./BNavText.vue_vue_type_script_setup_true_lang-7plzjrw7.mjs";
import { _ as _36, a as a18, b as b5, c as c5 } from "./BNavbarToggle.vue_vue_type_script_setup_true_lang-Ghpgh-1m.mjs";
import { _ as _37 } from "./BOverlay.vue_vue_type_script_setup_true_lang-lqljGFn8.mjs";
import { _ as _38 } from "./BPagination.vue_vue_type_script_setup_true_lang-nX8CXZzX.mjs";
import { _ as _39, a as a19, b as b6, c as c6, d as d4 } from "./BPlaceholderWrapper.vue_vue_type_script_setup_true_lang-M02BgK6i.mjs";
import { _ as _40 } from "./BPopoverOrchestrator.vue_vue_type_script_setup_true_lang-1QxERE4A.mjs";
import { _ as _41, a as a20 } from "./BProgress.vue_vue_type_script_setup_true_lang-CsDufGcF.mjs";
import { _ as _42 } from "./BSpinner.vue_vue_type_script_setup_true_lang-NE8IkK7C.mjs";
import { _ as _43, a as a21, b as b7, c as c7, d as d5, e as e3, f as f3, g } from "./BTable.vue_vue_type_script_setup_true_lang-BA0mZ4UT.mjs";
import { _ as _44 } from "./BTableSimple.vue_vue_type_script_setup_true_lang-CFZul_nj.mjs";
import { _ as _45, a as a22 } from "./BTabs.vue_vue_type_script_setup_true_lang-Dv8cL29B.mjs";
import { B as B3, _ as _46 } from "./BToastOrchestrator.vue_vue_type_style_index_0_lang-H5BEfz-H.mjs";
import { _ as _47 } from "./BTooltip.vue_vue_type_script_setup_true_lang-C0_nPddw.mjs";
import { useBreadcrumb } from "./src/composables/useBreadcrumb/index.mjs";
import { useColorMode } from "./src/composables/useColorMode/index.mjs";
import { useModal } from "./src/composables/useModal/index.mjs";
import { useModalController } from "./src/composables/useModalController/index.mjs";
import { useScrollspy } from "./src/composables/useScrollspy/index.mjs";
import { useToastController } from "./src/composables/useToastController/index.mjs";
import { usePopoverController } from "./src/composables/usePopoverController/index.mjs";
import { vBColorMode } from "./src/directives/BColorMode/index.mjs";
import { vBToggle, vBToggle as vBToggle2 } from "./src/directives/BToggle/index.mjs";
import { vBPopover } from "./src/directives/BPopover/index.mjs";
import { vBScrollspy } from "./src/directives/BScrollspy/index.mjs";
import { vBTooltip } from "./src/directives/BTooltip/index.mjs";
import { b as b8, B as B4, a as a23 } from "./classes-IC0yVJlq.mjs";
import { B as B5 } from "./BAlert-BhnJeO-F.mjs";
import { B as B6 } from "./BDropdown-CczkXkK7.mjs";
import { B as B7 } from "./BOffcanvas-XGubijFO.mjs";
import { B as B8 } from "./floatingUi-BJ4wPPsf.mjs";
import { bootstrapPlugin } from "./src/plugins/bootstrap/index.mjs";
import { breadcrumbPlugin } from "./src/plugins/breadcrumb/index.mjs";
import { createBootstrap } from "./src/plugins/createBootstrap/index.mjs";
import { modalControllerPlugin } from "./src/plugins/modalController/index.mjs";
import { modalManagerPlugin } from "./src/plugins/modalManager/index.mjs";
import { popoverPlugin } from "./src/plugins/popoverController/index.mjs";
import { rtlPlugin } from "./src/plugins/rtl/index.mjs";
import { toastPlugin } from "./src/plugins/toastController/index.mjs";
const index = /* @__PURE__ */ Object.freeze(/* @__PURE__ */ Object.defineProperty({
  __proto__: null,
  componentNames,
  componentsWithExternalPath,
  composableNames,
  composablesWithExternalPath,
  directiveNames,
  directivesWithExternalPath
}, Symbol.toStringTag, { value: "Module" }));
export {
  _ as BAccordion,
  a as BAccordionItem,
  B5 as BAlert,
  _2 as BAvatar,
  a2 as BAvatarGroup,
  _3 as BBadge,
  _4 as BBreadcrumb,
  a3 as BBreadcrumbItem,
  _5 as BButton,
  _6 as BButtonGroup,
  a4 as BButtonToolbar,
  _8 as BCard,
  a5 as BCardBody,
  b as BCardFooter,
  _9 as BCardGroup,
  c as BCardHeader,
  d as BCardImg,
  e as BCardSubtitle,
  a6 as BCardText,
  f as BCardTitle,
  _10 as BCarousel,
  a7 as BCarouselSlide,
  _7 as BCloseButton,
  _12 as BCol,
  _11 as BCollapse,
  _13 as BContainer,
  B6 as BDropdown,
  _15 as BDropdownDivider,
  a8 as BDropdownForm,
  b2 as BDropdownGroup,
  c2 as BDropdownHeader,
  d2 as BDropdownItem,
  e2 as BDropdownItemButton,
  f2 as BDropdownText,
  _16 as BForm,
  _19 as BFormCheckbox,
  a11 as BFormCheckboxGroup,
  _17 as BFormDatalist,
  _20 as BFormFile,
  a9 as BFormFloatingLabel,
  _21 as BFormGroup,
  _22 as BFormInput,
  _18 as BFormInvalidFeedback,
  _24 as BFormRadio,
  a12 as BFormRadioGroup,
  a10 as BFormRow,
  _25 as BFormSelect,
  _26 as BFormSelectOption,
  a13 as BFormSelectOptionGroup,
  _27 as BFormSpinbutton,
  a14 as BFormTag,
  _28 as BFormTags,
  b3 as BFormText,
  _29 as BFormTextarea,
  c3 as BFormValidFeedback,
  _30 as BImg,
  _23 as BInput,
  _31 as BInputGroup,
  a15 as BInputGroupText,
  _32 as BLink,
  _33 as BListGroup,
  a16 as BListGroupItem,
  B2 as BModal,
  _34 as BModalOrchestrator,
  _35 as BNav,
  a17 as BNavForm,
  b4 as BNavItem,
  c4 as BNavItemDropdown,
  d3 as BNavText,
  _36 as BNavbar,
  a18 as BNavbarBrand,
  b5 as BNavbarNav,
  c5 as BNavbarToggle,
  B7 as BOffcanvas,
  _37 as BOverlay,
  _38 as BPagination,
  _39 as BPlaceholder,
  a19 as BPlaceholderButton,
  b6 as BPlaceholderCard,
  c6 as BPlaceholderTable,
  d4 as BPlaceholderWrapper,
  B8 as BPopover,
  _40 as BPopoverOrchestrator,
  _41 as BProgress,
  a20 as BProgressBar,
  _14 as BRow,
  _42 as BSpinner,
  _45 as BTab,
  _43 as BTable,
  a21 as BTableLite,
  _44 as BTableSimple,
  a22 as BTabs,
  b7 as BTbody,
  c7 as BTd,
  d5 as BTfoot,
  e3 as BTh,
  f3 as BThead,
  B3 as BToast,
  _46 as BToastOrchestrator,
  _47 as BTooltip,
  g as BTr,
  B as BootstrapVueNextResolver,
  b8 as BvCarouselEvent,
  B4 as BvEvent,
  a23 as BvTriggerableEvent,
  i4 as Components,
  i as Composables,
  i2 as Directives,
  i6 as Plugins,
  i5 as Resolvers,
  index as Types,
  i3 as Utils,
  bootstrapPlugin,
  breadcrumbPlugin,
  componentNames,
  componentsWithExternalPath,
  composableNames,
  composablesWithExternalPath,
  createBootstrap,
  directiveNames,
  directivesWithExternalPath,
  modalControllerPlugin,
  modalManagerPlugin,
  popoverPlugin,
  rtlPlugin,
  toastPlugin,
  useBreadcrumb,
  useColorMode,
  useModal,
  useModalController,
  usePopoverController,
  useScrollspy,
  useToastController,
  vBColorMode,
  vBToggle as vBModal,
  vBPopover,
  vBScrollspy,
  vBToggle2 as vBToggle,
  vBTooltip
};
