// bootstrap-vue-next/dist/keys-BZewGSMQ.mjs
const genericBvnPrefix = "BootstrapVueNext__";
const withBvnPrefix = (value, suffix = "") => {
  const suffixWithTrail = `${suffix}___`;
  return `${genericBvnPrefix}ID__${value}__${suffix ? suffixWithTrail : ""}`;
};
const createBvnInjectionKey = (name) => withBvnPrefix(name);
const createBvnPluginInjectionKey = (name) => withBvnPrefix(`${name}__plugin`);
const carouselInjectionKey = createBvnInjectionKey("carousel");
const tabsInjectionKey = createBvnInjectionKey("tabs");
const progressInjectionKey = createBvnInjectionKey("progress");
const listGroupInjectionKey = createBvnInjectionKey("listGroup");
const avatarGroupInjectionKey = createBvnInjectionKey("avatarGroup");
const accordionInjectionKey = createBvnInjectionKey("accordion");
const checkboxGroupKey = createBvnInjectionKey("checkboxGroup");
const radioGroupKey = createBvnInjectionKey("radioGroup");
const collapseInjectionKey = createBvnInjectionKey("collapse");
const globalShowHideStorageInjectionKey = createBvnPluginInjectionKey("globalShowHideStorage");
const dropdownInjectionKey = createBvnInjectionKey("dropdown");
const navbarInjectionKey = createBvnInjectionKey("navbar");
const rtlPluginKey = createBvnPluginInjectionKey("rtl");
const breadcrumbGlobalIndexKey = `${genericBvnPrefix}global_breadcrumb`;
const breadcrumbPluginKey = createBvnPluginInjectionKey("breadcrumbPlugin");
const modalManagerPluginKey = createBvnPluginInjectionKey("modalManager");
const defaultsKey = createBvnPluginInjectionKey("defaults");
const inputGroupKey = createBvnInjectionKey("inputGroup");
const buttonGroupKey = createBvnInjectionKey("buttonGroup");
const toastPluginKey = createBvnPluginInjectionKey("toast");
const modalControllerPluginKey = createBvnPluginInjectionKey("modalController");
const popoverPluginKey = createBvnPluginInjectionKey("popover");
const formGroupPluginKey = createBvnInjectionKey("formGroupPlugin");
export {
  breadcrumbPluginKey as a,
  breadcrumbGlobalIndexKey as b,
  collapseInjectionKey as c,
  defaultsKey as d,
  modalManagerPluginKey as e,
  accordionInjectionKey as f,
  globalShowHideStorageInjectionKey as g,
  avatarGroupInjectionKey as h,
  buttonGroupKey as i,
  checkboxGroupKey as j,
  formGroupPluginKey as k,
  radioGroupKey as l,
  modalControllerPluginKey as m,
  inputGroupKey as n,
  listGroupInjectionKey as o,
  popoverPluginKey as p,
  progressInjectionKey as q,
  rtlPluginKey as r,
  tabsInjectionKey as s,
  toastPluginKey as t,
  dropdownInjectionKey as u,
  carouselInjectionKey as v,
  withBvnPrefix as w,
  navbarInjectionKey as x
};
